pandas/io/pytables.py

```diff
@@ -284,7 +284,19 @@ def to_hdf(
         f(path_or_buf)
 
 
-def read_hdf(path_or_buf, key=None, mode: str = "r", **kwargs):
+def read_hdf(
+    path_or_buf,
+    key=None,
+    mode: str = "r",
+    errors: str = "strict",
+    where=None,
+    start: Optional[int] = None,
+    stop: Optional[int] = None,
+    columns=None,
+    iterator=False,
+    chunksize: Optional[int] = None,
+    **kwargs,
```

Comment:
a solution to this would be to remove kwargs and instead add a keyword ``backend_kwargs={}`` or ``driver_kwargs={}``

Reply:
that would work, will need deprecation cycle, and im all about _removing_ deprecations these days

Reply:
for now ill need to make a quick PR reverting the `assert not kwargs` this introduced